snakemake_calling_files/sorted_T5_sample_AI-69_S60_flagstat.log

```diff
@@ -0,0 +1,16 @@
+1400000 + 0 in total (QC-passed reads + QC-failed reads)
```

Comment:
Here you have the alignment logs, is that how it should be? It's not colling, hah. In any case, the results should be the same I guess and a separate plus that you showed these logs. They can be useful in bioinf to assess the quality of alignment, etc. So they're lightweight and adding them is the right thing to do, I'll put a plus. Unless, then the folder is named differently :)